IDEO.md

```diff
@@ -0,0 +1,34 @@
+<img src="./etc/image/OKG_Badge_600x600.png" width="200" align="right"/>
+
+# Integrated Development Environment for Ontologies
+
+# Table of contents
+
+* [Use case development](#use-case-development)
+
+
+# Introduction
+
+EDMC OKG presents the Integrated Development Environment for Ontologies (IDEO)... It is based on and is a generalizaton of FIBO BTDM methodology descibed in [FIBO_BTDM.docx](./etc/site/fibo/doc/FIBO_BTDM.docx)
```

Comment:
We should take care to distinguish what's part of the FIBO ontology and what is part of the EDMC offering. Like stripping out the EDMC website content and Vue code.